packages/website/CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # @brisk-docs/website
 
+## 0.8.0
+
+### Minor Changes
+
+- 017909a: Remove the possibility of specifying docsList/packagesPaths from config.
+
+  Before this change, specifying `docsList` or `packagesPaths` would override
+  the intended configuration of the `docs`/`packages` entries.
+
+  - 7014ef8: Allow homepage tile images to be customised
```

Comment:
did i mess something up here? this looks nested - but shouldn't be?

Reply:
I think this is a bug in our changelog generator function which I should sort out but haven't yet.

Partly because testing this is annoying as heck.